packages/repository-modernizer/src/util/constants.js

```diff
@@ -65,9 +65,9 @@ module.exports = {
 
     NON_ADOBE_DEPENDENCIES: "nonadobedependencies",
 
-    BUNDLE: "bundle",
+    BUNDLE: ["bundle"],
 
-    CONTENT_PACKAGE: "content-package",
+    CONTENT_PACKAGING_TYPES: ["content-package", "pom"],
```

Comment:
Please create a new constant if required, and keep the old CONTENT_PACKAGE, it is expected to be of string type (used in log msg [here](https://github.com/adobe/aem-cloud-service-source-migration/blob/master/packages/repository-modernizer/src/create-base-project-structure.js#L459)).